tf-modules/ingest/lambda-functions.tf

```diff
@@ -127,8 +127,8 @@ resource "aws_lambda_function" "message_consumer" {
   handler          = "index.handler"
   role             = var.lambda_processing_role_arn
   runtime          = "nodejs16.x"
-  timeout          = 100
-  memory_size      = 256
+  timeout          = lookup(var.lambda_timeouts, "messageConsumer", 100)
```

Comment:
Can you increase the default to 300?  DAAC reports both memory and timeout issues.

Reply:
just for `messageConsumer`